README.md

```diff
@@ -19,8 +19,35 @@ Simple, quick to set up stock notification bot for Nvidia 3080 that I used to ge
 3. pip install any dependencies you don't have
 	-  `pip install twilio`
 	- `pip install selenium`
-	- `pip install win10toast`
+	- `pip install win10toast` (Windows 10 users only)
 ## How to Run
-`python notifier.py`
+
+### Windows
+
+```
+python notifier.py
+```
+
+### MacOS
+
+#### If Python 3 is installed as `python` on your path:
+
+```
+python notifier-mac.py
+```
+
+#### If Python 3 is installed as `python3` on your path:
+
+```
+python3 notifier-mac.py
```

Comment:
Since you merged the two scripts, this should say notifier.py instead of notifier-mac.py